<!-- 
    This launch file launches everything necessary to run auton code in the simulator.
 -->
<launch>
    <arg name="rvizconfig" default="$(find mrover)/config/rviz/auton_sim.rviz"/>
    <arg name="world_name" default="world"/>
    <arg name="use_odom" default="true"/>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true"/>

    <!-- launch rover core nodes -->
    <include file="$(find mrover)/launch/rover_core.launch"/>

    <!-- launch the simulator -->
    <include file="$(find mrover)/launch/simulator.launch">
        <arg name="world_name" value="$(arg world_name)"/>
    </include>

    <!-- launch auton -->
    <!-- <include file="$(find mrover)/launch/auton.launch">
        <arg name="sim" value="true"/>
        <arg name="use_ekf" value="true"/>
        <arg name="ekf_start_delay" value="10.0"/>
    </include> -->
</launch>